SUASSystem/SUASSystem/sda_converter.py

```diff
@@ -157,6 +169,7 @@ def get_uav_avoid_coordinates(self):
         for xy_loc_point in self.current_path:
             gps_points.append(inverse_haversine(self.initial_coordinates, xy_loc_point))
 
+        #return gps_points
```

Comment:
Please remove the commented out code if it is not needed.

Reply:
done